db/seeds.d/020-provisioning_templates_list.rb

```diff
@@ -98,13 +113,16 @@ def seeded_templates
     end
 
     def kinds
-      kinds = {}
-      TemplateKind.default_template_labels.keys.map(&:to_sym).each do |type|
-        kinds[type] = TemplateKind.unscoped.find_by_name(type)
-        kinds[type] ||= TemplateKind.unscoped.create(:name => type)
-        raise "Unable to create template kind: #{format_errors kinds[type]}" if kinds[type].nil? || kinds[type].errors.any?
+      @kinds ||= begin
+        kinds = {}
+        TemplateKind.default_template_labels.keys.map(&:to_sym).each do |type|
+          kind ||= TemplateKind.unscoped.find_or_create_by(name: type)
+          kind.description = template_description[kind.name]
+          kind.save!
+          kinds[type] = kind
+        end
+        kinds
       end
-      kinds
     end
```

Comment:
This block has been moved to `db/seeds.d/020-provisioning_templates_list.rb` in d689adb5ebc2e93d1fa9944642d08a71a71f931b so you need to rebase on top of that and put your changes there. Easiest way is to make a copy of this file, then rebase and "use deleted file" and then change the new file manually.